src/ipc-handlers.ts

```diff
@@ -216,6 +217,99 @@ export async function updateSite(
 	return mergeSiteDetailsWithRunningDetails( userData.sites );
 }
 
+export async function connectWpcomSite(
+	event: IpcMainInvokeEvent,
+	site: SyncSite,
```

Comment:
Made changes in https://github.com/Automattic/studio/pull/610/commits/05e94314ad5c87144aa010fe921cde49a83be942 to address this. I think it makes sense to have this so that there is a consistency between these two functions